tf_mnist/training.py

```diff
@@ -82,9 +83,10 @@ def feed_dict(train):
 
         if i % 10 == 0:
             # Record summaries and test-set accuracy
-            summary, acc = sess.run([merged, accuracy], feed_dict=feed_dict(False))
+            summary, acc = sess.run([merged, accuracy, loss], feed_dict=feed_dict(False))
             test_writer.add_summary(summary, i)
             print(json.dumps({'step': i, 'accuracy': acc.item()}))
+            print(json.dumps({'step': i, 'loss': acc.item()}))
```

Comment:
Shouldn't this be `loss.item()`? Also, in order for Valohai's metadata machinery to correctly chart this, it should be merged to the same print:

```suggestion
            print(json.dumps({'step': i, 'accuracy': acc.item(), 'loss': loss.item()}))
```

Reply:
sorry this was a stupid test, I am just evaluating valohai, I realized I cannot use my own repo, please ignore